tests/sources/fixtures/confluence/fixture.py

```diff
@@ -7,26 +7,50 @@
 """
 import io
 import os
-import random
-import string
+import time
 
 from flask import Flask, request
 
-DATA_SIZE = os.environ.get("DATA_SIZE", "small").lower()
-_SIZES = {"small": 1000000, "medium": 2000000, "large": 6000000}
-FILE_SIZE = _SIZES[DATA_SIZE]
-LARGE_DATA = "".join([random.choice(string.ascii_letters) for _ in range(FILE_SIZE)])
+from tests.commons import WeightedFakeProvider
+
+fake_provider = WeightedFakeProvider()
+
+DATA_SIZE = os.environ.get("DATA_SIZE", "medium")
```

Comment:
missing a `.lower()`?

Reply:
also, intentional that the default went from `"small"` to `"medium"`?

Reply:
Good catch! Will do `.lower()`.

As about data_size - all defaults should be consistent, so swapped here to "medium".

Our nightly and on-commit tests use explicit DATA_SIZE anyway, so it's the change for local testing without providing DATA_SIZE.